tests/autoyast/installation.pm

```diff
@@ -96,6 +101,15 @@ sub run {
                 @needles = grep { $_ ne 'autoyast-confirm' } @needles;
                 $confirmed = 1;
             }
+            elsif (match_has_tag('autoyast-license') || match_has_tag('inst-betawarning')) {
```

Comment:
I rewrote it to be more explicit about BETA=1 - but functionality did not change, please check
I also tested without BETA=1 parameter and test fails, as expected.